Modules/gcmodule.c

```diff
@@ -1405,6 +1411,36 @@ gc_is_tracked(PyObject *module, PyObject *obj)
     return result;
 }
 
+PyDoc_STRVAR(gc_freeze__doc__,
+"freeze() -> None\n"
+"\n"
+"Freeze all current tracked objects and ignore them for future collections.\n"
+"This can be used before a fork to make the gc copy-on-write friendly.\n"
+"Note: collection before a fork may free pages for future allocation\n"
+"which can cause copy-on-write.\n"
+);
+
+static PyObject *
+gc_freeze(PyObject *module)
```

Comment:
We'd prefer if you used ArgumentClinic for those functions. It's very easy to use: https://docs.python.org/3/howto/clinic.html